client/gutenberg/extensions/subscriptions/edit.jsx

```diff
@@ -0,0 +1,69 @@
+/** @format */
+
+/**
+ * External dependencies
+ */
+import { Component } from '@wordpress/element';
+import { TextControl, FormToggle } from '@wordpress/components';
+
+/**
+ * Internal dependencies
+ */
+import { __ } from 'gutenberg/extensions/presets/jetpack/utils/i18n';
+import { sprintf } from '@wordpress/i18n/build/index';
+
+class SubscriptionEdit extends Component {
+	render() {
+		const { attributes, className, isSelected, instanceId, setAttributes } = this.props;
+		const {
+			subscribe_placeholder,
+			subscribe_button,
+			success_message,
+			show_subscribers_total,
+		} = attributes;
+
+		if ( isSelected ) {
+			return (
+				<div className={ className }>
+					{ __( 'Show total subscribers' ) }
+					<FormToggle
+						checked={ show_subscribers_total }
+						onChange={ () => {
+							setAttributes( { show_subscribers_total: ! show_subscribers_total } );
+						} }
+					/>
+					<TextControl placeholder={ subscribe_placeholder } required onChange={ () => {} } />
+					<input type="button" disabled value={ __( 'Subscribe' ) } />
```

Comment:
The disabled button here could be confusing - it could make the user think they need to do something to make it active. 

Would we want to make it non-disabled, but add an `onClick` handler that prevents its default behavior?

Reply:
Do use the button component instead of defining an `<input />` element. This improves accessibility and is the orthodox way of doing this.

Example:

```
<Button isDefault={ true }>{ __( 'Subscribe' ) }</Button>
```

The component can be imported in line 7, as a part of `@wordpress/components`.

Reply:
See also: https://wordpress.org/gutenberg/handbook/components/button/